src/devtools/client/inspector/markup/components/Node.js

```diff
@@ -174,6 +180,7 @@ class Node extends PureComponent {
           role: "treeitem",
           onMouseEnter: this.onMouseEnter,
           onMouseLeave: this.onMouseLeave,
+          ref: shouldScrollIntoView ? this.scrollIntoView : undefined,
```

Comment:
This is a neat usage of `ref`.

Reply:
Thanks. One interesting aspect is that the callback will only get called once when it is first set, on subsequent renders it will not get called - and that is important, because otherwise, it would keep scrolling to this node on every render...